tekton/resources.yaml

```diff
@@ -37,7 +37,7 @@ spec:
 apiVersion: tekton.dev/v1alpha1
 kind: PipelineResource
 metadata:
-  name: tekton-bucket-nightly
+  name: pipeline-tekton-bucket-nightly
```

Comment:
@afrittoli Wasn't sure about this one as it doesn't seem to use `/pipeline` :sweat_smile:

Reply:
This is not really used, the resource is created by the trigger template